src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.cs

```diff
@@ -215,7 +215,10 @@ private static bool IsRenameValid(MutableConflictResolution conflictResolution,
             try
             {
                 var projectOpt = conflictResolution.CurrentSolution.GetProject(renamedSymbol.ContainingAssembly, cancellationToken);
-                if (renamedSymbol.ContainingSymbol.IsKind(SymbolKind.NamedType))
+
+                // For tuple fields the ContainingAssembly is System.ValueTuple, so we can't get a project
```

Comment:
```suggestion
                // For tuple fields the ContainingType is System.ValueTuple, so we can't get a project
```

Reply:
_Well, actually_ ContainingAssembly is correct, but I should have put `.dll` on the end though to make it clearer, so will re-word this.

Reply:
You know what, I can just put the tuple check at the top of the method and be done with it.. much easier.